HopsanGUI/HcomHandler.cpp

```diff
@@ -1490,6 +1499,36 @@ void HcomHandler::executeAddPlotRightAxisCommand(const QString cmd)
     changePlotVariables(cmd, 1, true);
 }
 
+void HcomHandler::executeChangeDiagramSizeCommand(const QString cmd)
+{
+    QStringList args = splitCommandArguments(cmd);
+
+    if (args.size() != 2 && args.size() != 4) {
+        HCOMERR("Wrong number of arguments in chds, should be 4");
+        return;
+    }
+
+    if (mpCurrentPlotWindow) {
+        bool ok;
```

Comment:
Ok is not checked so invalid values like 'dog' will be converted to zero and accepted.
Actually I think that the input args should be evaluated, then it would be possible to control size programatically in scripts. I have added such code before in other commands... or maybe it was in functions... not sure.

Reply:
Also what happens if size or position is negative.

Reply:
Good points, I have added SymHop evaluation and sign checks. There also seems to be a built-in check in Qt that prevents moving the window outside the screen.

We should probably add evaluation in all HCOM commands that takes numerical arguments.